data/transactions/logic/eval_test.go

```diff
@@ -35,6 +35,9 @@ import (
 	"github.com/algorand/go-algorand/protocol"
 )
 
+var defaultEvalProto = config.ConsensusParams{LogicSigVersion: 1, LogicSigMaxCost: 20000}
+var defaultEvalParams = EvalParams{Proto: &defaultEvalProto, Trace: &strings.Builder{}, Txn: &transactions.SignedTxn{}}
```

Comment:
Trace should be nil unless set; otherwise the one big global trace for all tests could grow to be huge